### PR TITLE
Maintain new/empty line consistency in docblocks.

### PR DESCRIPTION
**Pull request by ADmad**

### CODE REVIEW

File: phpcs.xml.dist [hunk 1]
```diff
@@ -3,4 +3,16 @@
     <config name="installed_paths" value="../../cakephp/cakephp-codesniffer" />
 
     <rule ref="CakePHP" />
+    <rule ref="SlevomatCodingStandard.Commenting.DocCommentSpacing">
```
Comment (ADmad) [inline phpcs.xml.dist]: This rule would actually be adding to our coding standard.

File: phpcs.xml.dist [hunk 2]
```diff
@@ -3,4 +3,16 @@
...
+        <properties>
+            <property name="linesCountBeforeFirstContent" value="0" />
+            <property name="linesCountBetweenDescriptionAndAnnotations" value="1" />
+            <property name="linesCountBetweenDifferentAnnotationsTypes" value="0" />
+            <property name="linesCountBetweenAnnotationsGroups" value="0" />
+            <property name="linesCountAfterLastContent" value="0" />
+        </properties>
+    </rule>
+    <rule ref="SlevomatCodingStandard.Namespaces.FullyQualifiedClassNameInAnnotation">
```
Comment (ADmad) [inline phpcs.xml.dist]: This is just temporary to deal with an issue in new slevomat coding standards release.

**Comment by othercorey** [comment]
We should merge the fixes.

File: phpcs.xml.dist (hunk 1 above)
Comment (markstory) [reply phpcs.xml.dist:6]: Does this new rule have a fixer?

Comment (ADmad) [reply phpcs.xml.dist:6]: Yes, all changes in this PR are auto fixed, no manual changes required.

**Pull request merged**